### DISCUSSION TITLE
Inquiry About Free Trial for Prime Figma UI Kit

### DISCUSSION THREAD

**Discussion by alexzerntev** [primefaces/community#3227] [Figma UI Kit] [answered]
Hello,

I am new to Figma and interested in purchasing the Prime Figma UI Kit. Since I am still exploring Figma's capabilities, I would like the opportunity to try the UI Kit beforehand to ensure it meets both my needs and those of my company.

Is it possible to access a free trial or preview version of the Prime Figma UI Kit?

Thank you!

**Comment by onursenture** [comment, marked as answer]
Hello, unfortunately, we do not offer a free trial. However, you can explore the UI Kit through the preview links:

[Light Theme](https://www.figma.com/design/G855HjuSyK8viJr0a5ZjRG/Preview-%7C%C2%A0PrimeOne-%7C-3.0.1?node-id=830-41631&t=m1MbOTTqKsBSRBLS-1) | [Dark Theme](https://www.figma.com/design/XBQzDl4vDOO0pyxEGYcICt/Preview-%7C%C2%A0Dark-%7C-PrimeOne-%7C-3.0.1?node-id=806-36648&t=7AME0kw905t3PVVY-1)

Note for future readers of this message: The links provided are for the UI Kit 3.0.1 Preview. Please check the UI Kit page in the showcases for the most up-to-date links.
